packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -204,6 +206,20 @@ class DropdownMenu extends StatefulWidget {
   /// The default width of the menu is set to the width of the text field.
   final MenuStyle? menuStyle;
 
+  /// Controls the text being edited or selected in the menu.
+  ///
+  /// If null, this widget will create its own [TextEditingController].
+  final TextEditingController? controller;
+
+  /// The callback is called when a selection is made or there are any changes in
+  /// the text field.
+  ///
+  /// {@macro flutter.widgets.editableText.onChanged}
+  ///
+  /// Defaults to null. By default, the menu shows up if there's any changes in the
+  /// text field.
+  final ValueChanged<String>? onChanged;
+
```

Comment:
I think it makes sense to have this function execute whenever the text changes, but I also think there are a lot of use cases, where people actually only care about when a possible value is selected.

For example if I do some logic in onChanged to handle the new selected value, but now it gets fired whenever the text changes, that is something I need to take special care of every time. And check if this value is actually an allowed value.
But in reality the only thing I care about in my application is what real possible value got selected and not what the user currently types into the text field.

Reply:
Changing to `onSelected` makes sense to me. I just updated the code. Thanks for the suggestion:)